src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8840,10 +8800,6 @@ private bool IsPossibleStatement(bool acceptAccessibilityMods)
                 case SyntaxKind.IdentifierToken:
```

Comment:
nit: this could just be how my brain works, I'm able to handle the `if` statements without braces, but `while` throws me off (no longer feels like a loop). Could we keep the braces here at least?